src/app/list/Survey.tsx

```diff
@@ -1,11 +1,106 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+
+interface Question {
+  id: number,
+  content: string,
+  choices: Choice[]
+}
+
+interface Choice {
+  id: number,
+  content: string,
+  type?: string,
+  rate?: number
+}
 
 function Survey() {
+  const questionList: Question[] = [
+    {
+      id: 1,
+      content: "is it a question?",
+      choices: [
+        {
+          id: 1,
+          content: "choice A",
+          rate: 10
+        },
+        {
+          id: 2,
+          content: "choice B",
+          rate: 5
+        },
+        {
+          id: 3,
+          content: "choice C"
+        }
+      ]
+    },
+    {
+      id: 2,
```

Comment:
Instead of having these mock questions inside the Survey component we should create json files for dumb data. For example `mockQuestions.json`. Hard coding those kind of things are makes code bad.